storage/replica.go

```diff
@@ -1660,3 +1677,49 @@ func (r *Replica) maybeAddToSplitQueue() {
 		r.store.splitQueue.MaybeAdd(r, r.store.Clock().Now())
 	}
 }
+
+// processReplicaCmd get a raft command from command queue and
+// call processRaftCommand to execute in a loop until there is
+// not any command in the queue for 5 seconds.
```

Comment:
maybe im missing something, but what is the point of shutting it down like this? seems perfectly fine to leave this goroutine running until the stopper shuts it down.